docs/runtime/v8js-metrics.md

```diff
@@ -0,0 +1,329 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: V8 JS Engine Runtime
+--->
+
+# Semantic Conventions for V8 JS Engine Runtime Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+This document describes semantic conventions for V8 JS Engine Runtime metrics in OpenTelemetry. This engine is used in some javascript runtime such as Node.js and Deno.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Experimental](#experimental)
+  - [Metric: `v8js.gc.duration`](#metric-v8jsgcduration)
+  - [Metric: `v8js.memory.limit`](#metric-v8jsmemorylimit)
+  - [Metric: `v8js.memory.used`](#metric-v8jsmemoryused)
+  - [Metric: `v8js.heap.space.available_size`](#metric-v8jsheapspaceavailable_size)
+  - [Metric: `v8js.heap.space.physical_size`](#metric-v8jsheapspacephysical_size)
+
+<!-- tocstop -->
+
+## Experimental
+
+**Status**: [Experimental][DocumentStatus]
+
+**Description:** Experimental V8 JS Engine Runtime metrics captured under `v8js`.
+
+### Metric: `v8js.gc.duration`
+
+This metric is [recommended][MetricRecommended].
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/metrics/api.md#instrument-advisory-parameters)
+of `[ 0.01, 0.1, 1, 10 ]`.
+
+<!-- semconv metric.veightjs.gc.duration(metric_table) -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `v8js.gc.duration` | Histogram | `s` | Garbage collection duration. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Could you add any recommendation/guidance of how to retrieve this value?

Reply:
done